src/tiled/command.cpp

```diff
@@ -105,6 +107,8 @@ Command Command::fromQVariant(const QVariant &variant)
         command.name = hash[namePref].toString();
     if (hash.contains(commandPref))
         command.command = hash[commandPref].toString();
+    if (hash.contains(shortcutPref))
+        command.shortcut = QKeySequence(hash[shortcutPref].toString());
```

Comment:
In the `QVariant` documentation, it says that it casts `QKeySequence` to `QString` automatically. Is this still required?